table/src/main/java/com/bazaarvoice/emodb/table/db/astyanax/TableJson.java

```diff
@@ -335,6 +357,17 @@ Delta newNextState(String storageUuid, StorageState state, Object markerValue) {
                 .build();
     }
 
+    // Delta for storing the Table metadata change updates.
+    Delta newTableMetadataUpdate(String tableName, String updateType, String time) {
```

Comment:
I'm not sure this method belongs here.  The delta isn't being written to the `TableJson` so why is this method in this class?